griptape/loaders/file_loader.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from os import PathLike
+from typing import TypeVar, Union, cast
+
+from attrs import Factory, define, field
+
+from griptape.artifacts import BaseArtifact
+from griptape.artifacts.blob_artifact import BlobArtifact
+from griptape.drivers import (
+    AudioParserDriver,
+    BaseFileManagerDriver,
+    BaseParserDriver,
+    CsvParserDriver,
+    LocalFileManagerDriver,
+    PillowParserDriver,
+    PyPdfParserDriver,
+    TextParserDriver,
+)
+from griptape.loaders import BaseLoader
+from griptape.utils import get_mime_type
+
+A = TypeVar("A", bound=BaseArtifact)
+
+
+@define
+class FileLoader(BaseLoader[Union[str, PathLike], bytes, A]):
+    file_manager_driver: BaseFileManagerDriver = field(
+        default=Factory(lambda: LocalFileManagerDriver(workdir=None)),
+        kw_only=True,
+    )
+    parser_driver_map: dict[str, BaseParserDriver] = field(
+        default=Factory(
+            lambda: {
+                "application/pdf": PyPdfParserDriver(),
```

Comment:
Wondering if this should just be `PdfParserDriver`? Do we care about exposing the fact that it uses pypdf?